### DISCUSSION TITLE
readme tag

### DISCUSSION THREAD

**Discussion by titouandk** [vuejs/rfcs#565] [Polls]
What about adding readme tags directly on VueJS components?

Could it be useful for the tooling to exploit?

```vue
<readme lang="md">
# My component

Hello **world**.
</readme>

<template></template>

<style scoped></style>

<script setup></script>
```

**Comment by posva** [comment]
It’s already possible to add custom blocks to SFC so they can be handled like vue-i18n does

**Reply by titouandk** [reply]
Interesting, thank you
